sql-plugin/src/main/scala/com/nvidia/spark/rapids/RegexParser.scala

```diff
@@ -1025,6 +1025,14 @@ class CudfRegexTranspiler(mode: RegexMode) {
         case 'b' | 'B' if mode == RegexSplitMode =>
           // see https://github.com/NVIDIA/spark-rapids/issues/5478
           throw new RegexUnsupportedException("word boundaries are not supported in split mode")
+        case 'b' | 'B' =>
+          previous match {
+            case Some(RegexEscaped(ch)) if "DWS".contains(ch) =>
```

Comment:
Do you know if this has to do word boundaries around negated character classes, or is it specifically `\D`, `\S`, `\W`?

Reply:
Good catch, pushing another fix

Reply:
Oh I just remembered, in that case this should also include `\H` and `\V`